execute/report/report_test.go

```diff
@@ -480,7 +480,7 @@ func Test_Builder_Build(t *testing.T) {
 		{
 			name: "half report",
 			args: args{
-				maxReportSize: 2529,
+				maxReportSize: 2654,
```

Comment:
where do we get these numbers from?

Reply:
The tests failed because the new FeeValueJuels field increased the size of the reports. These tests failed with the new size of the reports, so I updated maxReportSize with these values.